src/VstsDemoBuilder/Controllers/EnvironmentController.cs

```diff
@@ -131,7 +131,7 @@ public JsonResult GetGroups()
                     _templates.Groups = new List<string>();
                     foreach (var group in templates.Groups)
                     {
-                        if (group.ToLower() != "private")
+                        //if (group.ToLower() != "private")
```

Comment:
If you don't need the "private" then you should probably clean up this code

Reply:
I was testing something, forgot to remove this. I'll clean up.